### PR TITLE
Remove local find_glob_pattern; use new version from filetools instead

### PR DESCRIPTION
**Pull request by kelseymh**
Depends on ~~[Framework PR #3297](https://github.com/easybuilders/easybuild-framework/pull/3297)~~; implemented based on comments from @mboisson in #2010 .

### CODE REVIEW

**Comment by kelseymh** [comment]
I _think_ the "No module named..." errors above are slightly misleading.  Lots of other EasyBlocks (.e.g, wrf.py, tensorflow.py, etc.) all import stuff from easybuild.tools.filetools successfully.  In both blender.py and root.py, the new line I added is

    from easybulld.tools.filetools import find_glob_pattern

This directive depends on having [Framework PR #3297](https://github.com/easybuilders/easybuild-framework/pull/3297) available.  So the failure is really not finding that function within the module.

**Comment by mboisson** [comment]
@kelseymh it is expected that this PR will fail until the other one gets merged.

**Comment by kelseymh** [comment]
Right!  I was more commenting on the error message itself, for which the `err` string, I think, is directly from Python.

File: easybuild/easyblocks/r/root.py [hunk 1]
```diff
@@ -29,12 +29,12 @@
 @author: Jens Timmerman (Ghent University)
 """
 from distutils.version import LooseVersion
-import glob
 import os
 
 from easybuild.framework.easyconfig import CUSTOM
 from easybuild.easyblocks.generic.cmakemake import CMakeMake
 from easybuild.tools.build_log import EasyBuildError
+from easybulld.tools.filetools import find_glob_pattern
```
Comment (boegel) [inline easybuild/easyblocks/r/root.py:37]: @kelseymh same typo here `bulld` -> `build`

File: easybuild/easyblocks/b/blender.py [hunk 2]
```diff
@@ -27,11 +27,11 @@
 
 @author: Samuel Moors (Vrije Universiteit Brussel)
 """
-import glob
 import os
 
 from easybuild.easyblocks.generic.cmakemake import CMakeMake
 from easybuild.tools.build_log import EasyBuildError
+from easybulld.tools.filetools import find_glob_pattern
```
[the remark this answers is not in the document]
Comment (kelseymh) [reply easybuild/easyblocks/b/blender.py:34]: Argh!  How did I not see that?

**Comment by boegel** [comment]
tested with existing `ROOT` and `Blender` easyconfigs, good to go, thanks @kelseymh!

**Pull request merged**